.github/workflows/kind.yml

```diff
@@ -62,8 +79,16 @@ jobs:
       uses: actions/download-artifact@v1
       with:
         name: antrea-ubuntu-cov
+      # TODO: Create path to two image artifacts when uploading artifacts, so multiple
+      # artifacts can be downloaded at once.
```

Comment:
I am not sure I understand this comment: do you mean bundle both images in a single artifact?

Reply:
I was looking at this: https://github.com/actions/download-artifact#download-all-artifacts

They still seem to be different artifacts.